T7Suite/AFRMap.cs

```diff
@@ -394,7 +393,7 @@ private float[] LoadTargetAFRMap(string filename, int size)
                         }
                     }
                 }
-                catch (Exception E)
+                catch (Exception)
```

Comment:
I think I rather want to have a logger.Debug() on these LoadTargetAFRMap() exceptions.